components/brave_ads/common/custom_notification_ads_features.h

```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_COMMON_CUSTOM_NOTIFICATION_ADS_FEATURES_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_COMMON_CUSTOM_NOTIFICATION_ADS_FEATURES_H_
+
+#include <string>
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+#include "brave/components/brave_ads/common/custom_notification_ads_constants.h"
+#include "build/build_config.h"
+
+namespace brave_ads::features {
+
+BASE_DECLARE_FEATURE(kCustomNotificationAds);
```

Comment:
If removing `::features` namespace, can ew please append Feature to the feature param